src/blog/article/PageContent.styles.tsx

```diff
@@ -1,31 +1,27 @@
 import styled from 'styled-components'
 import { ShareBar } from '../../components/shareBar'
-import { space } from '../../theme/space'
+import { lg, xxl } from '../../theme/space'
 import { mediaQueries } from '../../theme/mediaQueries'
 
 export const ShareBarWrapper = styled.div`
   display: none;
   position: absolute;
-  top: -${space.xxl};
-  left: ${space.lg};
-  margin-top: 100px;
+  top: -${xxl}px;
```

Comment:
is there a reason you need to use px unit instead of the rem from the space object?